README.md

```diff
@@ -1,59 +1,34 @@
-# Tendermint
+# LazyLedger Core
 
-![banner](docs/tendermint-core-image.jpg)
+![banner](docs/lazyledger-banner.png)
 
-[Byzantine-Fault Tolerant](https://en.wikipedia.org/wiki/Byzantine_fault_tolerance)
-[State Machines](https://en.wikipedia.org/wiki/State_machine_replication).
-Or [Blockchain](<https://en.wikipedia.org/wiki/Blockchain_(database)>), for short.
+[![Go version](https://img.shields.io/badge/go-1.14-blue.svg)](https://github.com/moovweb/gvm)
+[![Telegram](https://img.shields.io/discord/669268347736686612.svg)](https://medium.com/lazyledger)
+[![license](https://img.shields.io/github/license/tendermint/tendermint.svg)](https://github.com/lazyledger/lazyledger-core/blob/master/LICENSE)
+[![](https://tokei.rs/b1/github/lazyledger/lazyledger-core?category=lines)](https://github.com/lazyledger/lazyledger-core)
```

Comment:
Do you mean this one @adlerjohn ? The service seam down (temporarily?). We can delete it:
```suggestion
```